### ISSUE TITLE
About CRN dataset

### ISSUE THREAD

**Issue by ZhiXia-SEU** [a4152684/KT-Net#2]
Thanks for you great work again， I also have two question in dataset.
 Are PCN and CRN the same data set?
 Are there any other differences between them besides the difference in total point cloud points？
Looking forward to your reply！

**Comment by a4152684** [comment]
Yes, I think these are the same dataset. As the paper says: "We only render the partial scans with one random virtual view instead of eight random views like PCN, hence the number of our training data is one eighth of PCN, but we keep the testing data the same with PCN."
